### PR TITLE
Change OSX 'activiation policy' when running headless mode.

### PR DESCRIPTION
**Pull request by frewsxcv**
...or also when outputting a file.

Fixes servo#10836.

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/10935)

### CODE REVIEW

**Comment by frewsxcv** [comment]
This might also fix servo#9513?

cc @jdm @larsbergstrom

**Comment by jdm** [comment]
This is missing an update for ports/cef/Cargo.lock.

File: ports/glutin/window.rs [hunk 1]
```diff
@@ -65,6 +66,22 @@ const LINE_HEIGHT: f32 = 38.0;
 
 const MULTISAMPLES: u16 = 16;
 
+#[cfg(target_os = "macos")]
+fn builder_with_platform_options(mut builder: glutin::WindowBuilder)-> glutin::WindowBuilder {
+    if opts::get().headless || opts::get().output_file.is_some() {
+        // Prevent the window from showing in Dock.app, stealing focus,
+        // or appaearing at all when running in headless mode or generating an
```
Comment (aneeshusa) [inline ports/glutin/window.rs]: spelling: appearing

**Comment by frewsxcv** [comment]
Fixed the typo

@bors-servo r=larsbergstrom

**Comment by pcwalton** [comment]
@bors-servo: r=larsbergstrom p=10

This makes bots run tests faster so a higher priority will help the queue clear out

**Comment by aneeshusa** [comment]
You also typo'd the commit message: activiation -> activation

**Comment by mbrubeck** [comment]
```
   Compiling glutin_app v0.0.1 (file:///home/servo/buildbot/slave/linux-dev/build/ports/glutin)
/home/servo/buildbot/slave/linux-dev/build/ports/glutin/window.rs:16:25: 16:41 error: unresolved import `glutin::os::macos::ActivationPolicy`. Could not find `macos` in `glutin::os` [E0432]
/home/servo/buildbot/slave/linux-dev/build/ports/glutin/window.rs:16 use glutin::os::macos::{ActivationPolicy, WindowBuilderExt};
                                                                                             ^~~~~~~~~~~~~~~~
/home/servo/buildbot/slave/linux-dev/build/ports/glutin/window.rs:16:25: 16:41 help: run `rustc --explain E0432` to see a detailed explanation
/home/servo/buildbot/slave/linux-dev/build/ports/glutin/window.rs:16:43: 16:59 error: unresolved import `glutin::os::macos::WindowBuilderExt`. Could not find `macos` in `glutin::os` [E0432]
/home/servo/buildbot/slave/linux-dev/build/ports/glutin/window.rs:16 use glutin::os::macos::{ActivationPolicy, WindowBuilderExt};
                                                                                                               ^~~~~~~~~~~~~~~~
/home/servo/buildbot/slave/linux-dev/build/ports/glutin/window.rs:16:43: 16:59 help: run `rustc --explain E0432` to see a detailed explanation
```

**Comment by frewsxcv** [comment]
Woops forgot to cfg gate the import

**Comment by frewsxcv** [comment]
`cfg` gated the import, fixed the typo in the commit message

@bors-servo: r=larsbergstrom p=10

**Comment by frewsxcv** [comment]
Updated lockfiles. Didn't know that cef required servo-glutin

@bors-servo: r=larsbergstrom p=10

**Comment by jdm** [comment]
I did point it out 6 hours ago :)

**Comment by frewsxcv** [comment]
> I did point it out 6 hours ago :)

I thought ports/glutin was the only "port" that required glutin, so I didn't bother trying to update it (I did update cocoa-rs). Though I should probably stop assuming these things :P

**Comment by frewsxcv** [comment]
Tidy fail

@bors-servo: r=larsbergstrom p=10

**Pull request merged**